tests/main/interfaces-bluez/task.yaml

```diff
@@ -13,6 +13,11 @@ environment:
     SNAP_NAME: bluez
 
 execute: |
+    if [ "$KARCH" != "x86_64" ]; then
```

Comment:
In the same line, it would be great to keep executing these tests on arm archs, they are working fine there so far.